### PR TITLE
chore(trackingplan): collecting violation metrics for tracking plan

### PR DESCRIPTION
**Pull request by Jayachand**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by achettyiitr — APPROVED** [review]

File: src/util/utils.js [hunk 1]
```diff
@@ -21,8 +24,29 @@ const responseStatusHandler = (status, entity, id, url) => {
   }
 };
 
+const sendViolationMetrics = (validationErrors, dropped, metaTags) => {
+  const vTags = {
+    "Unplanned-Event": 0,
+    "Additional-Properties": 0,
+    "Datatype-Mismatch": 0,
+    "Required-Missing": 0,
+    "Unknown-Violation": 0,
```
Comment (abhimanyubabbar) [inline src/util/utils.js:33]: Can we use enums or constants as keys in this object ? The error type is this and can change in future, let's hardcode them as constants ?

**Review by abhimanyubabbar — DISMISSED** [review]
Minor nit.

File: src/util/utils.js (hunk 1 above)
Comment (Jayachand) [reply src/util/utils.js:32]: matching with violation-types
to support line `vTags[error.type] += 1;`

Comment (Jayachand) [reply src/util/utils.js:32]: This will be removed after two weeks.

**Review by abhimanyubabbar — APPROVED** [review]

**Review by achettyiitr — APPROVED** [review]

**Pull request merged**